hal/src/thumbv7em/clock/v2.rs

```diff
@@ -178,12 +638,11 @@ impl<T, N: PrivateDecrement> PrivateDecrement for Enabled<T, N> {
 
 impl<T, N: Counter> Counter for Enabled<T, N> {}
 
-/// Test
-pub fn test() {
+#[allow(dead_code)]
+fn test() {
```

Comment:
Yes, that was the plan. But it's nice to have in the code for now, because Ruat Analyzer catches errors in real code immediately. Maybe there's a way to continuously check doc code as well?

Reply:
Doc code can be checked with `no_run` as was said in other comments, boilerplate is a difficult issue to solve unfortunately.

Reply:
Closing this. We will delete the `test` function before release. I'm keeping it for now, because rust-analyzer doesn't point out failing doc examples.